README.md

```diff
@@ -30,19 +30,42 @@ server.plugin.register({
           }
          ]
       }],
-      mongo: require("mongodb")
+      mongo: require("mongodb"),
+      connectionOptions: {
```

Comment:
`connectionOptions` should be provided for each db (a sibling of `connectionString`, and not be a 'global' configuration. Some connection strings might point to a replica set, some other might not.